install_files/ansible-base/roles/tor-hidden-services/tasks/install_tor.yml

```diff
@@ -37,7 +37,15 @@
     apt-cache policy tor | sed -e 's/^\s*Installed:\ \(\S*\)/\1/g;tx;d;:x'
   changed_when: false
   register: extract_tor_version
-  when: "'amazon' in ansible_product_version"
+
+# Ubuntu upstream repositories serve a version of tor that is . Since
```

Comment:
This comment trails off - maybe "a version of tor that is very old"?

Otherwise I agree with the logic here... now stepping through the test plan